packages/nibijs/docs/classes/CustomChain.md

```diff
@@ -8,11 +8,11 @@ based on its chain ID.
 **`Example`**
 
 ```ts
-export const TEST_CHAIN = new CustomChain({
+export const chain = new CustomChain({
   prefix: "nibiru",
   shortName: "testnet",
   number: 1,
-}) // v0.19.2
+}) // v0.21.43
```

Comment:
Can we just remove this?

Reply:
Done e4d449a6009827dbb999b414a40c9b269948b43e